ONSdigital/takeon-egestion
# Feature spp2891 new contract

# Description
snapshot contract change

# How to test?
env: spp2891

To get 'new' contract change
Set the DEFAULT_SCHEMA to 'false' in lambda.tf
pass the following message into the 'spp-es-takeon-egestion-spp2891-db-export-input' queue
check if the correct file has been generated in the 'spp-es-takeon-egestion-spp2891-export' bucket

# Links
https://collaborate2.ons.gov.uk/jira/browse/SPP-2891
